src/drivers/intel/cavs/ssp.c

```diff
@@ -322,7 +323,7 @@ static inline int ssp_set_config(struct dai *dai,
 	sscr2 |= (ssp->params.quirks & SOF_DAI_INTEL_SSP_QUIRK_PSPSRWFDFD) ?
 		SSCR2_PSPSRWFDFD : 0;
 
-#if defined(CONFIG_ICELAKE)
+#if CAVS_VER == CAVS_VER_2_0
```

Comment:
the clocks are not defined by the IP but by the SOC integration, so this is not quite correct.

Reply:
@plbossart 
I've reverted changes in ssp.c